app/webpacker/components/RegistrationsV2/Register/CompetingStep.jsx

```diff
@@ -350,6 +337,9 @@ export default function CompetingStep({
                   {i18n.t('registrations.update')}
                 </Button>
                 <ButtonOr />
+                <Button secondary onClick={() => nextStep()}>
+                  {i18n.t('competitions.registration_v2.register.view_registration')}
+                </Button>
```

Comment:
The naming scheme here feels unintuitive to me. The button to **view** your registration (according to the i18n text, at least) internally invokes `nextStep` which in my mind is a mutation. So a button that touts to "view" actually "mutates". Am I missing something here?

Reply:
no mutation happening, the next step from the Update page is the View registration page